src/interactionManager/credentialOfferFlow.ts

```diff
@@ -161,4 +162,12 @@ export class CredentialOfferFlow extends Flow<
       }
     })
   }
+
+  public getOfferDisplay() {
+    return this.state.offerSummary.map((oc, idx) => {
+      const claim = this.state.issued[idx]?.claim
+      const credType = new CredentialType(oc.type, oc.credential || {} as CredentialDefinition)
+      return credType.display(claim)
+    })
+  }
```

Comment:
They do slightly different things, because one of them (`CredentialKeeper.display`) retrieves the credential definition from storage, while this one (`CredentialOfferFlow.getOfferDisplay`) uses the definition that's in the offer token in this interaction.
It can be improved perhaps by making `CredentialKeeper.display` take an optional `credDefinition`, or otherwise look it up from storage. Then we can use that method here on the `offerSummary` entries

Reply:
and also one is sync and one is async :sweat_smile: